projects/05/ComputerAdd-external.out

```diff
@@ -0,0 +1,15 @@
+| time |reset|RAM16K[0]|RAM16K[1]|RAM16K[2]|
```

Comment:
I don’t think you meant to add all these `*.out` files.

Reply:
Aren't the `*.out` files the test files? I've been adding them for each HDL. I think I shouldn't have added the `Rect` ones though, because the output for that one is seen on the screen.

Reply:
Oh, sorry, for some reason I hadn’t noticed you were adding them all. They’re the output that’s generated by the hardware simulator when the test scripts run, which is compared against the corresponding `*.cmp` files to check that the HDL is behaving correctly. So there’s no real need to add them, but it also doesn’t do any harm, so if you intended to do it then that’s all good.